app/Repositories/User/UserInterface.php

```diff
@@ -51,4 +51,94 @@ public function attachRole($role);
      * @return mixed
      */
     public function getUsersForRoleScope($roleId);
+
+      /**
+    * attach defect to user
+    * @param $user
+    * @param $defectId
+    * @throws \Exception
+    */
+    public function attachDefectToUser($user,$defectId);
```

Comment:
Didn't rename this function. Please pay close attention to replace all occurences